test/repos/dep_versioning_between/requirements.txt

```diff
@@ -0,0 +1 @@
+testfarhan>0.3,<0.5
```

Comment:
Do we already have a test where there is no `requirements.txt` file? I assume we probably do in some other test that imports a dep, but just checking.

Reply:
Not implicitly, but we can add it. It will download whatever the latest version of the dependency is.

Reply:
👍 I think it'd be useful to have one just for coverage sake.